tests/task_comments/task_comments_base_setup.py

```diff
@@ -0,0 +1,18 @@
+from tests.tasks.tasks_base_setup import TasksBaseTestCase
+from app.database.models.mentorship_relation import MentorshipRelationModel
+from app.database.models.tasks_list import TasksListModel
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+from app.utils.enum_utils import MentorshipRelationState
+from tests.test_data import user1, user2, user4
+from app.api.dao.task_comment import TaskCommentDAO
+
+class TaskCommentsBaseTestCase(TasksBaseTestCase):
+    def setUp(self):
+        super(TaskCommentsBaseTestCase, self).setUp()
```

Comment:
Are we still supporting Python 2? We should drop it even if we are.

If we are just supporting Python 3, this can just be:

```
super()
```

Feel free to open a new issue for this and the others

Reply:
AFAIK this repository is Python3-only.

Reply:
Hey, I will open an issue for that.